mpmp/utilities/data_utilities.py

```diff
@@ -331,6 +331,84 @@ def load_purity(mut_burden_df,
     return purity_df.loc[:, ['status', 'DISEASE', 'log10_mut']]
 
 
+def load_survival_labels(cancer_type,
+                         mut_burden_df,
+                         sample_info_df,
+                         verbose=False):
+    """Load data relevant to survival prediction.
+
+    Arguments
+    ---------
+    mut_burden_df (pd.DataFrame): dataframe with sample mutation burden info
+    sample_info_df (pd.DataFrame): dataframe with sample cancer type info
+    verbose (bool): if True, print verbose output
+
+    Returns
+    -------
+    survival_df (pd.DataFrame): dataframe where the "status" attribute is
+                                the relevant survival duration
+    """
+
+    if verbose:
+        print('Loading survival info and covariates...', file=sys.stderr)
+
+    # TODO: drop relevant NA columns later
```

Comment:
Is this TODO something you wanted to do before the paper goes out, or just something to look at eventually?

Reply:
Neither, this was a TODO I added during development but forgot to take out when I addressed it. The code on lines 396-398 should handle all the NAs, so I'll remove the TODO.